teams/migration/README.md

```diff
@@ -11,4 +11,5 @@ See [membership.json](membership.json)
 * [ticdc](https://github.com/pingcap/ticdc)
 * [dm](https://github.com/pingcap/dm)
 * [dumpling](https://github.com/pingcap/dumpling)
+* [tidb/br](https://github.com/pingcap/tidb/tree/master/br)
```

Comment:
that's not a repo 🤔. and `dumpling` is also inside tidb repo now.

Reply:
> that's not a repo 

Perhaps it should be described as "Code Locations" as before.

@zhangyangyu For the `membership.json` file, maybe we can define another field named `packages`?

Reply:
I'd like to change repo to "Code Locations". And in `membership.json`, currently it's more like for authority and privileges. I'd like leave it unchanged first.